mailing_list/management/commands/activate_all_lists.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import csv
+import logging
+import operator
+
+from django.core.management.base import BaseCommand, CommandError
+
+from lti_emailer.canvas_api_client import get_courses_for_account_in_term
+from mailing_list.models import MailingList
+
+
+OUTPUT_FIELDS = ['id', 'sis_course_id', 'name', 'course_code']
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Activates all mailing lists for a given account and enrollment term'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--account-id', type=int, required=True,
+                            help='Canvas account id')
+        parser.add_argument('--enrollment-term-id', type=int, required=True,
+                            help='Canvas enrollment term id')
+        parser.add_argument('--output-file', type=argparse.FileType('w'),
+                            help='File to output mailing list details to')
+
+    def handle(self, *args, **options):
+        try:
+            courses = get_courses_for_account_in_term(
+                          options['account_id'], options['enrollment_term_id'])
+        except RuntimeError as e:
+            raise CommandError(str(e))
+
+        course_lists = {}
+        failures = {}
+        defaults = {'access_level': MailingList.ACCESS_LEVEL_MEMBERS}
+        for course in courses:
+            try:
+                lists = MailingList.objects.\
+                            get_or_create_mailing_lists_for_canvas_course_id(
+                                    course['id'], defaults=defaults)
+            except RuntimeError as e:
+                failures[course['id']] = str(e)
+                continue
+            course_lists[course['id']] = [ml for ml in lists
+                                              if ml['is_primary_section']]
```

Comment:
I'm returning just primary sections because the initial request from HKS was for a list of the whole-course mailing lists by course.  But you're right, that's not well-documented, and may not make sense for all schools.

I'll put the secondary lists in another column.